src/test/kotlin/graphics/scenery/tests/examples/basic/DFTExample.kt

```diff
@@ -0,0 +1,101 @@
+package graphics.scenery.tests.examples.basic
+
+import org.joml.Vector3f
+import graphics.scenery.*
+import graphics.scenery.backends.Renderer
+import graphics.scenery.numerics.Random
+import graphics.scenery.textures.Texture
+import graphics.scenery.utils.Image
+import graphics.scenery.volumes.Colormap
+import graphics.scenery.volumes.TransferFunction
+import graphics.scenery.volumes.Volume
+import net.imglib2.type.numeric.integer.UnsignedByteType
+import java.nio.file.Paths
+import kotlin.concurrent.thread
+import kotlin.math.pow
+
+/**
+ * <Description>
+ *  Visualizes a DFT snapshot.
+ *
+ * @author Lenz Fiedler <l.fiedler@hzdr.de>
+ */
+class DFTExample : SceneryBase("DFTExample", wantREPL = System.getProperty("scenery.master", "false").toBoolean()) {
```

Comment:
You can remove the wantREPL stuff here, I had unfortunately forgotten to remove it from TexturedCubeExample after a recent change, sorry for the confusion.

Reply:
Done!